### ISSUE TITLE
Auth0 don't play nice with unsecure origins

### ISSUE THREAD

**Issue by jeliasson** [redwoodjs/playground-auth#14]
I'm trying to run `playground-auth` out of the canary box but Auth0 tells me that it must run from a secure origin. Anyone know how we can ignore this stuff while doing local (and unsecure) development?

For now, I have just removed the Auth0 references in `web/pages/HomePage` to get pass it. 

![image](https://user-images.githubusercontent.com/865493/101154975-cc596580-3626-11eb-973f-7d5f7c47f47a.png)

/cc @dthyresson @peterp

**Comment by dthyresson** [comment]
@jeliasson I have not seen this before, though I tend not to use the `window.location.href` but instead set `http://localhost:8910,` as that is what is setup in Auth0 as an allowed callback for the app's config.

I'll try to reproduce and report back.

If in fact Auth0 requires https may need to do port forwarding -- or perhaps we add some logic to hide Auth0 in dev when you don't intend to dev or test is.

Ie - enable flags per AuthProvider or something like that.

**Comment by jeliasson** [comment]
@dthyresson I think I have narrowed it down to a WSL 2 issue. Basically, before I did some tweaking on my side, I accessed the `playground-auth` using http://192.168.110.229:8910 (my WSL 2 ip-address) which then threw this error. Now after using Windows `netsh` to port forward `localhost:8910` to `192.168.110.229:8910` it is working fine again. I'm going to close this issue and will write a post about it in the forums once I have ironed out a few details.

/cc @Tobbe
